docs/com.bitwarden.ciphers.md

```diff
@@ -21,6 +21,8 @@ The `com.bitwarden.ciphers` doctype is used to store secret things for the
 - `login`: {object} - an object only used when type is 1 (see below)
 - `data`: {object} - a map of encrypted properties on the cipher
 - `fields`: {array} - a list of objects, with `type`, `name`, and `value` (encrypted)
+- `organization_id`: the identifier of a [bitwarden organization](./com.bitwarden.organization.md),
+- `collection_id`: "8869e3ee461551cc2bc4d5d9a107d0c1",
```

Comment:
```suggestion
- `collection_id`: {uuid} - the identifier of the collection,
```

Reply:
Woups, I still have my usaved tab in VS code, sorry for that 😨